src/test/codegen/debug-limited.rs

```diff
@@ -0,0 +1,28 @@
+// Verify that the limited debuginfo option emits llvm's FullDebugInfo, but no type info.
+//
+// ignore-windows
```

Comment:
Why aren't we running these tests on Windows? They don't seem to be doing anything platform specific that wouldn't work on Windows.

Reply:
Ah, that is a leftover from  the test I based it off of `debug-column.rs`, I'll remove it.

Reply:
Thanks!